### ISSUE TITLE
Websocket connection failures 

### ISSUE THREAD

**Issue by jbrannst** [openshift/openshift-ansible-contrib#98]
I get various connection failures when browsing the OSE web console. I have no issues on the command line console. Shutting down the other masters 02 and 03 solves the problem, but I assume it should work in HA as well. 

Maybe the asynchronous loading fails when the request is loadbalanced to a different master than the one the first page was served with. This is after accepting permanently all certificates. I am running Firefox 49.0.2 and I have seen this issue before. I noticed a mention in an internal doc about configuring reverse proxy, maybe there is some additional AWS config I must do manually that I may have missed?

**Comment by cooktheryan** [comment]
I believe we had resolved this in the past with the Cross-Zone Load Balancing: Enabled setting on the load balancers. Spinning up a cluster now

**Comment by cooktheryan** [comment]
@jbrannst so this is issue seems to only happen with firefox and an untrusted cert. I tested on my local machine with firefox and ran into a similar issue. I imported the ca.crt in firefox and the error goes away.  I'm wondering if this error relates back to a previous firefox bug where firefox would ask you to accept the certificate over and over without ever accepting the certificate. 

This does not seem to happen with chrome
